src/utils/hooks/useGetResponsesAndElementState.tsx

```diff
@@ -108,8 +102,11 @@ const useGetResponsesAndElementState = (props: { serialNumber: string }) => {
     // const isValid = evaluateExpression(element.validation, evaluationParameters)
     const results = await Promise.all([isEditable, isRequired, isVisible])
     const evaluatedElement = {
-      code: element.code,
       id: element.id,
+      code: element.code,
+      title: element.title,
+      elementTypePluginCode: element.elementTypePluginCode,
+      section: element.section as number,
```

Comment:
its not really part of this PR, but was wondering if we should put the response value here too, and was wondering why we would need 3 types  of return rather then just elementsState, in this hook.

Reply:
Not sure Andrei, I think we need to add this improvement on another issue really. I'm also not sure if we need all 3 objects, but so far they have been in use, so let's try to tidy up more as we go.

Reply:
cools